libp2p/network/network_test.go

```diff
@@ -13,7 +13,8 @@ import (
 	"github.com/pokt-network/pocket/shared/crypto"
 )
 
-const testIP6ServiceURL = "2a00:1450:4005:802::2004:8080"
+// https://www.rfc-editor.org/rfc/rfc2732
+const testIP6ServiceURL = "[2a00:1450:4005:802::2004]:8080"
```

Comment:
Great catch @0xBigBoss! :100: :heart: 

I'm suggesting the RFC which obsoletes the one you referenced (including the section reference).

```suggestion
// https://www.rfc-editor.org/rfc/rfc3986#section-3.2.2
const testIP6ServiceURL = "[2a00:1450:4005:802::2004]:8080"
```